claim/claim.go

```diff
@@ -7,10 +7,16 @@ import (
 	"time"
 
 	"github.com/oklog/ulid"
+	"github.com/pkg/errors"
 
 	"github.com/cnabio/cnab-go/bundle"
+	"github.com/cnabio/cnab-go/utils/schemaversion"
 )
 
+// DefaultSchemaVersion represents the schema version of the Claim
+// that this library returns by default
+const DefaultSchemaVersion = schemaversion.SchemaVersion("v1.0.0-WD")
```

Comment:
hmm, gosh I just realized that since we are still in working draft there's no way to detect breaking changes since the version stays the same until we get out of WD. 🤦‍♀ This version schema is so very unhelpful...

I was originally expecting that we could bump this to v1.0.1-WD but I think since we never released 1.0 that isn't right. We should ask around.

To be clear, we still need this change, but it may not help for this particular breaking change due to the weak versioning protocol...